backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/createbpartner/CreateBPartnerV2_StepDef.java

```diff
@@ -95,6 +125,7 @@ public void verify_bPartner_was_created_for_externalIdentifier_v2(@NonNull final
 			assertThat(persistedBPartner.getLanguage()).contains(language);
 			assertThat(persistedBPartner.getCode()).isEqualTo(code);
 			assertThat(persistedBPartner.getPhone()).isEqualTo(phone);
+			assertThat(persistedBPartner.getStorageWarehouse()).isEqualTo(storageWarehouse);
```

Comment:
when touching assertions - in particular when adding a number of them - please use the occasion to switch to soft-assertions